[2022-09-17 05:50:53,505] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-09-17 05:50:53,518] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-09-17 05:50:53,518] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 05:50:53,519] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-17 05:50:53,519] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 05:50:53,537] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-04-02 06:00:00+00:00
[2022-09-17 05:50:53,545] {standard_task_runner.py:52} INFO - Started process 4393 to run task
[2022-09-17 05:50:53,551] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2019-04-02T06:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp0lmfmdv7', '--error-file', '/tmp/tmphj2aepq2']
[2022-09-17 05:50:53,553] {standard_task_runner.py:77} INFO - Job 18: Subtask local_to_gcs_task
[2022-09-17 05:50:53,644] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [running]> on host 10d063af5ba7
[2022-09-17 05:50:53,709] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-17 05:50:53,738] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T06:00:00+00:00
[2022-09-17 05:50:57,523] {python.py:175} INFO - Done. Returned value was: None
[2022-09-17 05:50:57,541] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20190402T060000, start_date=20220917T055053, end_date=20220917T055057
[2022-09-17 05:50:57,579] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-17 05:50:57,639] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-09-20 15:32:11,807] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-09-20 15:32:11,835] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-09-20 15:32:11,835] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 15:32:11,835] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-20 15:32:11,835] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 15:32:11,857] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-04-02 06:00:00+00:00
[2022-09-20 15:32:11,866] {standard_task_runner.py:52} INFO - Started process 406 to run task
[2022-09-20 15:32:11,878] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2019-04-02T06:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpv0j5z8s8', '--error-file', '/tmp/tmptjmy36v0']
[2022-09-20 15:32:11,880] {standard_task_runner.py:77} INFO - Job 60: Subtask local_to_gcs_task
[2022-09-20 15:32:12,075] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [running]> on host 788dccad41b9
[2022-09-20 15:32:12,173] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-20 15:32:12,212] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T06:00:00+00:00
[2022-09-20 15:32:13,481] {python.py:175} INFO - Done. Returned value was: None
[2022-09-20 15:32:13,496] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20190402T060000, start_date=20220920T153211, end_date=20220920T153213
[2022-09-20 15:32:13,537] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-20 15:32:13,578] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-09-21 01:33:28,684] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-09-21 01:33:28,701] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [queued]>
[2022-09-21 01:33:28,701] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-21 01:33:28,701] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-21 01:33:28,701] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-21 01:33:28,720] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2019-04-02 06:00:00+00:00
[2022-09-21 01:33:28,728] {standard_task_runner.py:52} INFO - Started process 27586 to run task
[2022-09-21 01:33:28,734] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2019-04-02T06:00:00+00:00', '--job-id', '238', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpv3048hr1', '--error-file', '/tmp/tmpcc0gyhwv']
[2022-09-21 01:33:28,736] {standard_task_runner.py:77} INFO - Job 238: Subtask local_to_gcs_task
[2022-09-21 01:33:28,828] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2019-04-02T06:00:00+00:00 [running]> on host 788dccad41b9
[2022-09-21 01:33:28,890] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-21 01:33:28,924] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2019-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-04-02T06:00:00+00:00
[2022-09-21 01:33:30,120] {python.py:175} INFO - Done. Returned value was: None
[2022-09-21 01:33:30,139] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20190402T060000, start_date=20220921T013328, end_date=20220921T013330
[2022-09-21 01:33:30,197] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-21 01:33:30,253] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
